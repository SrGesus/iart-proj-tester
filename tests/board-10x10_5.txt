VB	LH	BC	LV	BD	FC	FD	VB	LH	VB
VE	VE	VC	FB	FE	VC	BB	VD	VB	LV
VD	BB	VC	BB	FB	LV	FC	VE	VD	LV
LV	VD	VD	VB	BD	BB	LV	BE	FD	LH
BB	LV	LV	FE	BE	VC	FC	VB	BC	FB
LV	VC	FC	VE	BC	VC	VC	FB	VC	VE
LV	LH	FB	VB	LH	VB	BC	VC	FC	BB
BD	BE	FE	FB	BD	LV	LH	FD	VC	BC
VC	LH	BD	FB	BB	BD	LH	LV	FC	LH
FB	LH	BB	FB	FC	FD	VE	BE	FE	FB
