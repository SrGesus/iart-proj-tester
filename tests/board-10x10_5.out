VB	LH	BB	LH	BB	FE	FB	VB	LH	VE
VD	VE	VD	FE	FC	VB	BE	VD	VE	LV
VB	BE	VB	BB	FE	LV	FC	VB	VC	LV
LV	VD	VC	VD	BB	BC	LH	BE	FB	LV
BD	LH	LH	FE	BD	VE	FB	VD	BE	FC
LV	VB	FE	VB	BE	VD	VC	FB	VD	VE
LV	LV	FD	VC	LV	VB	BB	VC	FD	BE
BD	BC	FE	FD	BE	LV	LV	FB	VB	BE
VD	LH	BB	FE	BD	BE	LV	LV	FC	LV
FD	LH	BC	FE	FC	FC	VD	BC	FE	FC
